### ISSUE TITLE
Constitution in Armenian generated by char-rnn

### ISSUE THREAD

**Issue by Hrant-Khachatrian** [dariusk/NaNoGenMo-2015#154]
We used Andrej Karpathy's [char-rnn](https://github.com/karpathy/char-rnn) to generate Constitution in Armenian with more than 55 000 words. 

```
Գ Լ ՈՒ Խ  1

ՍԱՀՄԱՆԱԴՐՈՒԹՅԱՆ ԸՆԴՈՒՆՈՒՄԸ, ՓՈՓՈԽՈՒՄԸ ԵՎ ՀԱՆՐԱՔՎԵՆ


Հոդված 115. Հանրապետության Նախագահի ընտրության երկրորդ փուլ, որին մասնակցելու իրավունք:
Յուրաքանչյուր ոք ունի իր ուղղությունների հետ կապված վեճերով:
Հոդված 67. Պատգամավորի համար Սահմանադրությամբ և օրենքներով սահմանված կարգով վարչական պատասխանատվության ենթարկելու հարց հարուցելու վերաբերյալ.
4) եզրակացություն է տալիս Հանրապետության Նախագահի կողմից նշանակված դատախազներին, կալանավորել հատուկ համաձայնագրով:
```

[Full version is here](https://github.com/YerevaNN/char-rnn-constitution/blob/master/samples/sample_bs50s512d0.6t0.5.txt).

Char-rnn is a character level language model based on recurrent neural networks. It is described in detail in Karpathy's [blog post](https://karpathy.github.io/2015/05/21/rnn-effectiveness/). 

All the details of our sample including some performance analysis are in [this blog post](http://yerevann.github.io/2015/11/12/generating-constitution-with-recurrent-neural-networks/) :)

**Comment by ikarth** [comment]
Non-English and topical! Nice addition to the generated library.
